### DISCUSSION TITLE
Documentation for contributors

### DISCUSSION THREAD

**Discussion by ghost** [lncrawl/lightnovel-crawler#2025] [Ideas]
I am moderately good with Python and I wanted to contribute to this project because I have been using it and I would like to improve it. To be more clear, I only used it when I didn't have wifi outside the home so I would download novels. My idea is that at this point their should be documentation or some kind of contributors guide so that if somebody wants to contribute they can get started as fast and as quickly as possible. Especially if they have no experience with selenium or beautifulsoup.

**Comment by dipu-bd** [comment]
There was a documentation was. It became outdated soon with the new changes, and was deleted it later. I had been planning to make a new one, but have not started it yet.

**Comment by ghost** [comment]
I see. Well, If you could make high-level documentation so that contributors who understand the code could fill in all the detailed documentation, I think that could work..
